ci-operator/config/codeready-toolchain/toolchain-e2e/codeready-toolchain-toolchain-e2e-master.yaml

```
@@ -41,7 +41,7 @@ tests:
     test:
     - as: test
       cli: latest
-      commands: make test-e2e CI_DISABLE_PAIRING=true
+      commands: make test-e2e CI_DISABLE_PAIRING=true DEPLOY_LATEST=true
```

based on the other PR, shouldn't this be just:
```suggestion
      commands: make test-e2e DEPLOY_LATEST=true
```

---

We still need `CI_DISABLE_PAIRING=true` because of the ksctl https://github.com/codeready-toolchain/toolchain-e2e/blob/master/make/ksctl.mk#L17

---

Gotcha, thanks. we could technically replace it with `DEPLOY_LATEST` too, but I'm not sure if Konflux doesn't rely on that in their CI. So it's fine to keep it as it is :+1: